concepts/stable-seekers/faq-stable.md

```diff
@@ -4,29 +4,35 @@ description: Common questions about getting and exchanging the Ppotocol's stable
 
 # FAQ - Stable Seekers and Holders
 
+## What is agEUR backed by?
+
+First, agEUR is backed by the collateral that was used to mint it. Then, if this collateral is volatile, HA can open positions on the collateral stored in the protocol to protect it from changes in collateral/agEUR exchange rate. In case this is not enough, SLPs come to over-collateralize the protocol.
+
 ## Could I use the protocol as a decentralized exchange?
 
 Angle can be used as a decentralized exchange where there is no slippage for swapping.
 
-If Angle's agEUR is backed by wETH and wBTC, a user wishing to swap wETH against wBTC with no slippage can choose to come to Angle, swap wETH against agEUR, and then use these agEUR to redeem wBTC.
+For example, if Angle's agEUR is backed by wETH and wBTC, a user wishing to swap wETH against wBTC with no slippage can choose to come to Angle, swap wETH against agEUR, and then use these agEUR to redeem wBTC.
```

Comment:
Can add something that says that Angle is just backed by

Reply:
What do you mean here?

Reply:
We should say: 

For example, if Angle's agEUR is backed by wETH and wBTC (at the moment Angle is only backed by USDC, DAI, FEI and FRAX), a user wishing to